writehat/templates/snippets/statusButtonGroup.html

```diff
@@ -0,0 +1,15 @@
+<div class="form-check form-check-inline">
+  <form id="status-radio">
```

Comment:
Should we store this form's selections in local storage so they persist across refreshes? We're doing something similar in several other places, like which nested components are collapsed in the report view or what filters are applied in the review status dashboard.

Reply:
Implemented in [https://github.com/blacklanternsecurity/writehat/pull/91/commits/e5ebc6ba0057d4835dd07f6541b8f48c3c7a98d8](https://github.com/blacklanternsecurity/writehat/pull/91/commits/e5ebc6ba0057d4835dd07f6541b8f48c3c7a98d8)